terraform/environments/mlra/modules/ALB/outputs.tf

```diff
@@ -0,0 +1,31 @@
+# output "athena_db" {
+#   value = aws_athena_database.lb-access-logs
+# }
+
+# output "security_group" {
+#   value = aws_security_group.lb
+# }
+
+# output "load_balancer" {
+#   value = aws_lb.loadbalancer
+# }
+
+# output "aws_lb_listener" {
+#   value = aws_lb.listener
+# }
+
+# output "aws_lb_target_group" {
+#   value = aws_lb.loadbalancer
+# }
```

Comment:
Is any of the above commented code required?

Reply:
They have been deleted